build.gradle

```diff
@@ -47,6 +47,14 @@ subprojects {
 
 apply from: 'cppSettings.gradle'
 
+def formatArgs = project.hasProperty('formatArgs') ? project.formatArgs : ''
+
+task format(type: Exec) {
+    workingDir "${rootDir}"
+    commandLine 'python'
+    args 'styleguide/format.py', formatArgs
```

Comment:
Just use the commandLine property: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:commandLine